src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -14,13 +14,19 @@
 (rf/reg-event-fx
  :wallet-connect/on-init-success
  (fn [{:keys [db]} [web3-wallet]]
-   {:db (assoc db :wallet-connect/web3-wallet web3-wallet)
-    :fx [[:dispatch [:wallet-connect/register-event-listeners]]]}))
+   {:db (assoc-in db [:wallet :wallet-connect :web3-wallet] web3-wallet)
```

Comment:
I think we should keep the :wallet-connect/... keys as roots and avoid nesting unless necessary. I remember there was a conversation at the offsite about this topic @J-Son89 @cammellos

Reply:
my point here is that it becomes pretty much a mess but interesting to hear other opinions

Reply:
Maybe we can have a session to discuss the db structure next week, but for now I'd propose to keep the structure with root keys, otherwise there's going to be some nasty conflicts. WDYT?

Reply:
i'd say it's best to change it to how it was to avoid unnecessary conflicts, since @shivekkhurana and me are already working with the structure we have on develop atm, but we can discuss and agree on a way during today's session 👍

Reply:
It's changed now